src/main/resources/com/mathworks/ci/TestResultsViewAction/index.jelly

```diff
@@ -0,0 +1,217 @@
+        </l:side-panel>
+
+        <l:app-bar title="MATLAB Test Results">
+            <button class="jenkins-button" onclick="window.open('https://matlab.mathworks.com/open/github/v1?repo=${GIT_URL}')" tooltip="View source in MATLAB Online">Open in MATLAB Online</button>
+            <t:help href="https://github.com/jenkinsci/matlab-plugin/blob/master/CONFIGDOC.md" tooltip="MATLAB Plugin Configuration Guide" />
+        </l:app-bar>
+        
+        <l:main-panel>
+            <button id="Total" onclick="showTests(id)" class="jenkins-button jenkins-!-margin-4 jenkins-!-padding-0" style="display: inline; width:8em;" tooltip="Total tests">
+                <p>${it.totalCount}<br></br><br></br>Total Tests</p>
+            </button>
+
+            <button id="Passed" onclick="showTests(id)" class="jenkins-button jenkins-!-margin-4 jenkins-!-padding-0" style="display: inline; width:8em;" tooltip="Passed tests">
+                <p>${it.passedCount}<br>
+                    <l:icon class="symbol-checkmark-circle-outline plugin-ionicons-api icon-lg jenkins-!-success-color"/>
+                </br><br></br>Passed</p>
+            </button>
+
+            <button id="Failed" onclick="showTests(id)" class="jenkins-button jenkins-!-margin-4 jenkins-!-padding-0" style="display: inline; width:8em;" tooltip="Failed tests">
+                <p>${it.failedCount}<br>
+                    <l:icon class="symbol-close-circle-outline plugin-ionicons-api icon-lg jenkins-!-error-color"/>
+                </br><br></br>Failed</p>
+            </button>
+
+            <button id="Incomplete" onclick="showTests(id)" class="jenkins-button jenkins-!-margin-4 jenkins-!-padding-0" style="display: inline; width:8em;" tooltip="Incomplete tests">
+                <p>${it.incompleteCount}<br>
+                    <l:icon class="symbol-alert-circle-outline plugin-ionicons-api icon-lg jenkins-!-warning-color"/>
+                </br><br></br>Incomplete</p>
+            </button>
+
+            <button id="NotRun" onclick="showTests(id)" class="jenkins-button jenkins-!-margin-4 jenkins-!-padding-0" style="display: inline; width:8em;" tooltip="Not run tests">
+                <p>${it.notRunCount}<br>
+                    <l:icon class="symbol-remove-circle-outline plugin-ionicons-api icon-lg"/>
+                </br><br></br>Not Run</p>
+            </button>
+
+            <j:forEach var="testSession" items="${it.testResults}" varStatus="status">
+                <table class="jenkins-table sortable" id="testResultsTable">
+                    <thead>
+                        <tr>
+                            <th class="pane-header" style="width: auto; min-width: 20em;">
+                                Test Name
+                            </th>
+                            <th class="pane-header" style="width: auto; min-width: 10em; text-align: right;">
+                                Duration (seconds)

```

Comment:
(s) instead of (seconds)?

Reply:
Is it better to include + and - signs or drop-down arrows instead?

Reply:
Updated (seconds) to (s)

> Is it better to include + and - signs or drop-down arrows instead?

I didn't get this, could you explain?

Reply:
Nothing major. Currently, it seems that you use the + and - buttons to expand and collapse sections in the report. I was wondering whether a drop-down arrow could do the same. In the test browser, we resort to the arrow for expansion: https://www.mathworks.com/help/matlab/matlab_prog/run-tests-using-test-browser.html#:~:text=You%20can%20also%20add%20tests%20to%20the%20test%20browser%20by%20clicking%20the%20drop%2Ddown%20arrow